python/py-sacremoses/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        alvations sacremoses 99102ce
+version             0.0.35
+revision            0
+name                py-sacremoses
+
+license             GPL-2.1
+maintainers         nomaintainer
+platforms           darwin
+
+description         Python port of Moses tokenizer, truecaser and normalizer
+
+long_description    ${description}
+
+homepage            https://github.com/alvations/sacremoses
+
+python.versions     27 37 38
+
+checksums           rmd160  fc932022c13a1028e5d854bc5513f18d7453f291 \
+                    sha256  6ea400212c2c8605fe80afec888a0d00f40c475d88a996294a04c719a42b2b4e \
+                    size    879729
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+}
```

Comment:
You have not added any of the required ```depends_lib```, please check upstream's ```setup.pyt```

Reply:
Done thanks.